app/controllers/api/v1/product_share_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::ProductShareController < Api::V1::BaseController
+    def create
+        product_share_email
+    end
+
+    def product_share_email
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.